shippable.yml

```diff
@@ -4,7 +4,7 @@ language: go
 
 matrix:
   include:
-    - go: 1.5.3
+    - go: 1.6.0
```

Comment:
@spxtr @ixdy 
Since we are no longer using shippable - can't we just remove this whole file?